src/app/common/utils/resource.js

```diff
@@ -364,7 +364,7 @@ function requestEpic(action$, store, { API }) { // FIXME API
     .mergeMap(function({ meta, payload }) {
       const { type, props, resource } = meta
 
-      const isListItem = !resource.item && resource.list && ['PATCH', 'PUT', 'DELETE'].includes(type)
+      const isListItem = !resource.item && resource.list && type !== 'GET'
```

Comment:
please add 'POST' in array instead.
request type may be 'OPTIONS', 'HEAD' and some custom request.